tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -3688,6 +3688,12 @@ pfcwd/test_pfcwd_all_port_storm.py::TestPfcwdAllPortStorm::test_all_port_storm_r
       - "https://github.com/sonic-net/sonic-mgmt/issues/21067 and 't0-56' in topo_type and asic_type in ['mellanox']"
       - "https://github.com/sonic-net/sonic-mgmt/issues/21082 and topo_name in ['t0-88-o8c80', 't1-48-lag', 't1-lag', 't1-32-lag', 't1-64-lag'] and asic_type in ['mellanox']"
 
+pfcwd/test_pfcwd_cli.py:
```

Comment:
xfail should be only for specific vendor

Reply:
Understood, thanks!